src/test/java/com/vinylteam/vinyl/service/impl/DefaultOneVinylOffersServiceTest.java

```diff
@@ -29,52 +31,53 @@
     @InjectMocks
     private DefaultOneVinylOffersService oneVinylService;
 
+    @Autowired
+    @MockBean
+    private OfferService mockedOfferService;
+
+    @Autowired
     @MockBean
-    private OfferService offerService;
+    private ShopService mockedShopService;
```

Comment:
no need to name ot mocked, just offerService
it is clear from @MockBean that it is mocked